source/mir/random/flex/internal/area.d

```diff
@@ -186,13 +190,13 @@ body
         }
         else if (iv.c == 1)
         {
-            area = S(0.5) * sh.a * intLength * (z + 2);
+            area = S(0.5) * sh.a * ivLength * (z + 2);
```

Comment:
On p.1 of the `Explanation` document, you have `beta = g'(x)`.  Do you mean `beta = g'(x0)` ... ?